cmd/kubectl-gadget/deploy.go

```diff
@@ -245,6 +246,9 @@ func init() {
 	deployCmd.PersistentFlags().StringSliceVar(
 		&allowedDigests,
 		"allowed-digests", []string{}, "List of allowed digests, if image digest is not part of it, execution will be denied")
+	deployCmd.PersistentFlags().StringSliceVar(
+		&allowedRegistries,
+		"allowed-registries", []string{}, "List of allowed registries, if gadgets does not come from one of it, execution will be denied")
```

Comment:
This implies that if the flag is empty, no gadgets will be executed since it is not in this whitelist. Maybe add comment that empty string means everything is allowed (As done with `allowed-digests`.